plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryLargeImage.php

```diff
@@ -214,4 +208,51 @@ private function get_open_dialog_directives( $block_context ) {
 	protected function get_block_type_script( $key = null ) {
 		return null;
 	}
+
+	/**
+	 * Returns an array of allowed HTML for wp_kses that includes directives.
+	 *
+	 * @return array
+	 */
+	private function get_allowed_directives_html() {
```

Comment:
Is this necessary? We should sanitize the HTML I think but we need to ensure relevant attribute data isn't removed. But this means maintaining a list of these attributes too.

Reply:
Based on my knowledge that should no longer be necessary as the directives should be allowed in sanitization since WP 6.5: https://core.trac.wordpress.org/ticket/61052. If it's needed then it's probably a bug.

Reply:
https://github.com/woocommerce/woocommerce/pull/55246/commits/dc87f25bb51ef91f37ba0322853247b5294d80d8